src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Validation/ValidationStack.cs

```diff
@@ -3,9 +3,8 @@
 
 using System.Collections.Generic;
 using System.Diagnostics;
-using Microsoft.AspNetCore.Mvc.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.ModelBinding.Internal
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Validation
 {
     public class ValidationStack
```

Comment:
What drives this one becoming truly `public`?

Reply:
It was exposed as a `protected` property on `ValidationVisitor`. My guess is that we should be safe to make the property `private` and the type `internal`, but let's sort this as part of https://github.com/aspnet/Mvc/issues/8694